Actors/CalculatorActor/build.ps1

```diff
@@ -0,0 +1,16 @@
+pushd src\CalculatorActorApplication\CalculatorActor.Interfaces\

```

Comment:
Does the script stop if any one command fail in the script ?

Reply:
No it currently does not do that.